### PR TITLE
📝  Add applied example to rangeinput extends

### PR DESCRIPTION
**Pull request by brianbancroft**
## Description
This PR proposes an example of how to document the `extends` option in the Grommet components that is less abstract and more applied, in support of #3389 

This PR only affects documentation, so no testing took place aside static checking. 

If the general idea is welcome, I would go and extend this to the other components in either this PR or in others.

### CODE REVIEW

**Comment by ShimiSun** [comment]
Thanks, @brianbancroft ! can you please push the results of the updated snapshots so we could see how the text was auto-generated? 
run `yarn test-update` and commit the snapshots that were created following these changes.
This should also fix the `jest` failures.

**Comment by brianbancroft** [comment]
@ShimiSun thanks for the direction. This is now done

File: src/js/components/RangeInput/doc.js [hunk 1]
```diff
@@ -47,7 +47,9 @@ export const themeDoc = {
   },
   'rangeInput.extend': {
     description: 'Any additional style for the RangeInput.',
-    type: 'string | (props) => {}',
+    type: `string | (props) => \`
+      color: #c9c19f;
```
Comment (ShimiSun) [inline src/js/components/RangeInput/doc.js]:
```suggestion
      color: #44444;
```
Choosing a color that is less distracting.

Comment (ShimiSun) [reply src/js/components/RangeInput/doc.js]: Or another idea is 
```suggestion
      any CSS styling;
```
so users won't be confused it is only for colors, thoughts?

Comment (brianbancroft) [reply src/js/components/RangeInput/doc.js]: Actually, the second one looks better. I'm going to have a little time on holiday monday, so I'll be sure to do that. 

Thanks!

**Comment by ShimiSun** [comment]
Thank you for your contribution and for helping the community with improving the docs!

**Pull request merged**